src/openrct2/interface/window.h

```diff
@@ -36,6 +36,7 @@ struct TitleSequence;
 struct TextInputSession;
 
 #define TEXT_INPUT_SIZE 1024
+#define TOP_TOOLBAR_CLEARANCE 28
```

Comment:
The bottom of the toolbar is at 27, strictly speaking.

*Sigh* Why is it always small stuff like this that gets so much discussion? Marijn's comment is a good example of getting bogged down. This makes creating PRs to clean up stuff a pain in the arse, to be quite honest.